docs/source/_templates/sdk_TP_tutorial.rst

```diff
@@ -1160,6 +1483,178 @@ the UTF-8 encoding of the string ``<a-entry>|<b-entry>|...``, where
         return strings.ToLower(hex.EncodeToString(hashBytes))
     }
 
+{% elif language == 'Rust' %}
+
+.. code-block:: rust
+    :caption: xo_rust/src/handler/state.rs XoState and Game structs/implementations
```

Comment:
Change the / to " and " 

(Reason: To be consistent with the previous caption; also, a tech-writing rule of thumb is to avoid using / in place of a word.)